### PR TITLE
Fix relations in MedicalEMR

### PR DESCRIPTION
**Pull request by lasley**
This is a continuation of #23 now that I know what I'm doing and didn't create a bunch of conflicts with merges and rebases on top of each other.

### CODE REVIEW

File: medical_emr/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Medical EMR is a multi-user, highly scalable, centralized Electronic
+Medical Record (EMR) for openERP.
+
+Medical provides a free universal Health Information System,
+so doctors and institutions all over the world,
+specially in developing countries will benefit from a centralized,
+high quality, secure and scalable system.
+
+Medical at a glance:
```
Comment (pedrobaeza) [inline medical_emr/README.rst]: This is the general explanation, not the module medical_emr one

**Comment by lasley** [comment]
@pedrobaeza  - That's a straight copy paste from the original module ( https://github.com/OCA/vertical-medical/blob/8.0/__unported__/oemedical_emr/__openerp__.py ). We're about to be switching up a lot of stuff in this incubator, I don't know if we should invest time in actually updating the ReadMe's at this time.

**Comment by lasley** [comment]
I went ahead and started a general todo's section on the wiki for this incubator and included Readmes. Most of the modules I've looked at just contain the general desc currently - https://github.com/OCA/vertical-medical/wiki/8.0-Incubator-To-Dos

**Comment by pedrobaeza** [comment]
I know that previously all modules contain the same description (because at the beginning, there was only big module), but you can now specialize this description for each module, although you expand it later with your changes.

**Comment by dreispt** [comment]
You need to keep the file history.
You should use `git mv` on the module in `__unported__` and then edit the files.

**Comment by lasley** [comment]
@pedrobaeza - I'll need to dig into this module more before I can write the ReadMe. At this point, I've really just been focused on getting them working so I can present an MVP to a client.

@dreispt - Whoops. Any ideas on fixing this retroactively?

**Comment by pedrobaeza** [comment]
OK, then remove the description and put only a brief summary ("Medical EMR addon for Odoo" or something)

**Comment by lasley** [comment]
No problem, thanks for the guidance!

File: medical_emr/__openerp__.py [hunk 2]
```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
-    'author': "Medical Team,Odoo Community Association (OCA)",
+    'author': "Medical Team, LasLabs, Odoo Community Association (OCA)",
```
Comment (Olivier-LAURENT) [inline medical_emr/__openerp__.py]: To keep consistency with master medical module, I propose: **Odoo** Medical Team, ...

File: medical_emr_data/__openerp__.py [hunk 3]
```diff
@@ -36,20 +37,10 @@
 
     'name': 'Medical EMR: Module Data',
     'version': '1.0',
-    'author': "OeMEdical Team,Odoo Community Association (OCA)",
+    'author': "OeMEdical Team, LasLabs, Odoo Community Association (OCA)",
```
Comment (Olivier-LAURENT) [inline medical_emr_data/__openerp__.py]: To keep consistency with master medical module, I propose: Odoo Medical Team, ...

**Comment by Olivier-LAURENT** [comment]
Just 2 minor details
:+1:

File: medical_emr/__openerp__.py [hunk 4]
```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
```
Comment (yvaucher) [inline medical_emr/__openerp__.py]: Please change versions to 8.0.1.0

Comment (lasley) [inline medical_emr/__openerp__.py]: Hi @yvaucher 

My understanding was that Odoo handles the major versioning, with the addons handling the minor versioning. Wouldn't we instead want `1.1`, which would present itself as `8.x.1.1` in Odoo?

Comment (nhomar) [inline medical_emr/__openerp__.py]: 2015-09-11 8:13 GMT-05:00 Yannick Vaucher notifications@github.com:

> Please change versions to 8.0.1.0

We asked to not ask for this rule until we have a clear lint, it well
generate a lot of discussione (unecesary)

## 

---

Saludos Cordiales

CEO at Vauxoo https://www.vauxoo.com Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
http://about.me/nhomar?promo=email_sig

File: medical_emr/__openerp__.py (hunk 2 above)
Comment (lasley) [inline medical_emr/__openerp__.py]: Agreed, I'll push a patch. Sorry I missed this comment!

File: medical_emr/__openerp__.py (hunk 4 above)
Comment (lmignon) [inline medical_emr/__openerp__.py]: @lasley  @nhomar The rule is now part of the official guideline https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers. @lasley Can you modify the version number according to the guidelines?

Comment (yvaucher) [inline medical_emr/__openerp__.py]: @lasley 8.0.1.1 is fine too

Comment (lasley) [inline medical_emr/__openerp__.py]: Cool thanks @yvaucher & @lmignon - Patch pushed

**Comment by lmignon** [comment]
@lasley Thank your for the changes :+1:

**Comment by yvaucher** [comment]
:+1: Just need a rebase to resolve conflict

**Comment by lasley** [comment]
Hah well that was interesting, brings a whole new meaning to merge conflicts when you're stepping them through time with a rebase. Good to go though!

**Comment by lasley** [comment]
I'm going to just rebase #38 on to this and close

**Comment by yvaucher** [comment]
@lasley You can rebase in same PR doing a `git push -f`

**Comment by lasley** [comment]
Ahhhhh so I should have rebased the other branch onto this one instead of the other way around. Got it, thanks for the advice @yvaucher !

**Pull request closed without merging**